docs/parts/conclusion.md

```diff
@@ -1,22 +1,24 @@
 <!-- See https://squidfunk.github.io/mkdocs-material/reference/ -->
 # Conclusion
 
-Thank you for participating in the workshop! Let's recap some of the key takeways that we've learned:
+Congratulations on completing this workshop! Let's recap some of the key takeways that we've learned:
 
-* ...
-* ...
-* ...
-
-## References
+* We've expanded our expertise by integrating AWS into our infrastructure provisioning workflow alongside Equinix Fabric using Pulumi. This integration showcases the versatility and power of Pulumi in orchestrating multi-cloud environments.
+* Through the creation of a multicloud environment utilizing Fabric Cloud Router, we've demonstrated the seamless connectivity between Google Cloud Platform (GCP) and Amazon Web Services (AWS), illustrating the potential for interconnected architectures across different cloud providers.
+* By automating the end-to-end deployment process using Pulumi, we've enhanced our infrastructure deployment efficiency and reliability, laying the groundwork for scalable and repeatable infrastructure management practices.
 
 ## Next Steps
 
+As you continue your journey, consider exploring further possibilities with Pulumi and multicloud architectures. Here are some suggested next steps:
+
+* Experiment with additional cloud providers and services to extend your multicloud environment. Integrate platforms like Microsoft Azure or other Equinix Fabric-supported clouds to diversify your infrastructure.
+* Dive deeper into Pulumi's features and capabilities, exploring advanced deployment strategies, infrastructure as code best practices, and leveraging Pulumi's extensive ecosystem of libraries and integrations.
+
 ## Resources
 
-Here are a few other resources to look at to continue your Equinix Metal journey:
+Here are a few other resources to look at to continue your Equinix Fabric journey:
 
 * [Deploy @ Equinix](https://deploy.equinix.com): A one-stop shop for blogs, guides, and plenty of other resources.
```

Comment:
I believe the Deploy site is Metal-specific (although it does have an external Fabric link in the header), but IMO keeping it is a good call (also a good call to remove the other links that are Metal-specific)

Reply:
Yeah that sounds good